about.html

```diff
@@ -62,320 +121,292 @@
-				I couldn't help but to wonder how many times have we traveled somewhere
-				but are not aware of what peoples lived and cultivated the land or what
-				structures they built to mold, shape, or even elevate the human
-				condition as we know it today?
-			</p>
-			<p>
-				Through these experiences, I realize how the past, present, and future
-				are part of an overlapping continuum of human experience - like deep
-				roots coming out of a mountain.
-			</p>
-			<p class="signature">ITLCT Co-Founder, Natasha Rabsatt</p>
-			<div class="btn-container">
-				<a class="button3" href="">Our Impact</a>
-			</div>
-		</section>
-		<!-- Footer -->
+		</div>
+	</div>
+	</section>
 
-		<footer>
-			<div class="wrapper">
-				<div class="footer-main">
-					<div class="footer-logo-btn">
-						<!-- logo -->
-						<a href="./index.html">
-							<img
-								src="./assets/icons/global/itlct-logo.svg"
-								alt="If These Lands Could Talk Logo"
-							/>
-						</a>
+	<section class="wrapper about-wrapper">
+		<h2 class="h4-styles">Our Story</h2>
+		<p>
+			The land has a way of teaching us if we want to know, if we want to
+			learn. "If These Lands Could Talk" started as an internal conversation
+			that emerged during travel across traditional Diné (or Navajo) homelands
+			in the summer of 2019. Driving through the red-pigmented and towering
+			mountains and canyons, I couldn't help but to wonder what these lands
+			would reveal if they had a voice. As I read through stories written by
+			local indigenous people accounting lessons from the past and the impact
+			that persists today - I learned that there are many gaps in education
+			and what is taught in traditional education or found in textbooks. As a
+			world traveler, I noticed parallels between these lands and other lands
+			that I had the privilege to travel to throughout my life. Much of our
+			world's history is passed down through generations, best and most
+			accurately at times told through the people who inhabited the land for
+			generations, and through the people who still nurture these lands today.
+		</p>
+		<p>
+			In a sudden moment, the words just came out of my mouth, "If These Lands
+			Could Talk, What Would They Say?" A question and statement all at once.
+			I couldn't help but to wonder how many times have we traveled somewhere
+			but are not aware of what peoples lived and cultivated the land or what
+			structures they built to mold, shape, or even elevate the human
+			condition as we know it today?
+		</p>
+		<p>
+			Through these experiences, I realize how the past, present, and future
+			are part of an overlapping continuum of human experience - like deep
+			roots coming out of a mountain.
+		</p>
+		<span class="signature">ITLCT Co-Founder, Natasha Rabsatt</span>
+		<div class="btn-container">
+			<a class="button3" href="/impact.html">Our Impact</a>
+		</div>
+	</section>
+	<!-- Footer -->
```

Comment:
Please update the footer. The good version is in index.html in the development branch.

Reply:
Added in updated footer